src/tab-panels.js

```diff
@@ -3,7 +3,12 @@ import React from 'react';
 
 export default function TabPanels({ accessibleId, activeIndex, children, className }) {
   return (
-    <div aria-labelledby={accessibleId} role="tabpanel" className={className}>
+    <div
+      aria-labelledby={accessibleId}
+      role="tabpanel"
+      className={className}
+      tabIndex={hasFocusableContent ? undefined : 0}
```

Comment:
I think exposing it on the API makes sense. Perhaps it can be more succinctly named: e.g. `focusContainer` ? But I am not sure about the payoff. I'm okay with running with this and awaiting feedback.

Reply:
That suggestion makes sense. The thing I like better about `hasFocusableContent` is that it gets at why it matters, whereas it's not clear from the name why someone would set `focusContainer` to true or false (unless they had a deep understanding of accessibility, or perused the docs).

In any case, thinking more about the naming here certainly makes sense.